gh_index_issues_to_dts_portal.py

```diff
@@ -78,30 +93,65 @@ def build_payload(project_records, project_issues):
     for issue in project_issues:  # iterate over gh issues
         pipeline = find_pipeline_by_issue(zenhub_metadata, issue.number)
 
+        last_comment_body = None
+        last_comment_date = None
+        comments = issue.get_comments()
```

Comment:
i was wondering if there is some way to avoid having to do this check for every project issue. like if it's closed, can we ignore updates to it? but obviously we would miss updates in that case.

this is slow, but not as slow as i would expect given how much data it's pulling down 👍

Reply:
I'll check with @amenity if its okay to ignore closed projects's comments

Reply:
@chiaberry @johnclary -What are the disadvantages of a slow script?